### PR TITLE
Removing old/outdated code and some small cleanup

### PR DESCRIPTION
**Pull request by oyssolbo**
After the update to the trilateration, a lot of old and useless code was left behind in the repo.

The following has been done:
-merged dsp_constants and parameters
-changed name of hydrophone-namespace and file to analyze_data
-changed from using pointers to arrays

### CODE REVIEW

**Comment by oyssolbo** [comment]
As mentioned in #24, the old code is saved locally in case we need to revert in the future

File: Makefile [hunk 1]
```diff
@@ -37,14 +37,14 @@ vortex_acoustics : $(OBJ) | $(DRIVER_ARCHIVE)
 	$(CXX) $(CFLAGS) $^ -o $@ $(LDFLAGS)
 
 $(BUILD_DIR) :
-	mkdir -p $@/driver
+	mkdir -p
```
Comment (chrstrom) [inline Makefile:40]: What does mkdir -p do when the directory name is left blank?

File: Source/analyze_data.cpp [hunk 2]
```diff
@@ -1,33 +1,69 @@
...
+ * Initializing the extern filter-variables
+ */
+float32_t state_coefficients[4 * num_stages] = 
+{
+        0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0
+};
+
+float32_t filter_coefficients[5 * num_stages] =
+{
+        0.56942484, 0.0, -0.56942484,             /* Numerator filter 1                               */
```
Comment (chrstrom) [inline Source/analyze_data.cpp:13]: Is there any documentation that explains how the filter coefficients are chosen?

Comment (oyssolbo) [reply Source/analyze_data.cpp:13]: Is in the headerfile parameters.h

File: Makefile (hunk 1 above)
Comment (oyssolbo) [reply Makefile:40]: Have no clue! Bug I didn't notice. Should revert this, however I am terrible with makefiles and cannot get one workinh

File: Source/analyze_data.cpp (hunk 2 above)
Comment (oyssolbo) [reply Source/analyze_data.cpp:13]: See parameters.h

Comment (chrstrom) [reply Source/analyze_data.cpp:13]: Nice :+1: Could you add a reference to that here? Just to make it easy to find for others

File: Makefile (hunk 1 above)
Comment (chrstrom) [reply Makefile:40]: Fair haha. Can still merge without it, before testing on the actual board starts :wink:

File: Source/analyze_data.cpp (hunk 2 above)
Comment (oyssolbo) [reply Source/analyze_data.cpp:15]: Done

File: Makefile (hunk 1 above)
Comment (oyssolbo) [reply Makefile:40]: The makefile is not that important as the code can be compiled elsewhere. It is nice to get working for testing the code on the computer etc.

**Review by chrstrom — APPROVED** [review]
lgtm :+1:

**Pull request merged**